src/cases/components/case-hearings-list/case-hearings-list.component.ts

```diff
@@ -53,6 +57,10 @@ export class CaseHearingsListComponent implements OnInit {
         this.hasReadOnlyAction = true;
       }
     }
+
+    this.featureToggleService.getValue(AppConstants.FEATURE_NAMES.enableHearingAmendments, false).subscribe((enabled) => {
```

Comment:
This can be be avoided if we can use the return type as Observable instead of subscribing